app/services/v1/henry_ford/verify_service.rb

```diff
@@ -19,7 +19,10 @@ def initialize(user, patient_params, session_id)
   attr_accessor :resource
 
   def call
-    if patient_params[:hfhs_patient_detail_id].present?
+    if patient_params[:id].present?
+      @resource = HfhsPatientDetail.find(patient_params[:id])
+      confirm_resource!
+    elsif patient_params[:hfhs_patient_detail_id].present?
```

Comment:
Why do we want to send the ID under `id` and once `hfhs_patient_detail_id`? Can we always send it under `hfhs_patient_detail_id`?